src/script/interpreter.h

```diff
@@ -207,8 +213,10 @@ bool VerifyScript(const CScript &scriptSig,
     ScriptError *error = NULL,
     unsigned char *sighashtype = NULL);
 
-// string prefixed to data when validating signed messages either via DATASIGVERIFY or RPC call.  This ensures
+// string prefixed to data when validating signed messages  via RPC call.  This ensures
```

Comment:
double space between `messages` and `via`.

Reply:
Done.